packages/graphiql/resources/renderExample.js

```diff
@@ -82,6 +82,7 @@ function graphQLFetcher(graphQLParams) {
   const api = isDev ? '/graphql' : 'https://swapi.graph.cool/';
```

Comment:
hey thanks for adding these!

as a final measure to confirm it works end to end, can you set the fetch URL to be fixed to the netlify URL in the other examples here? then this will close the reference ecosystem, as you will see the demo using that schema url in the deploy preview for this PR, the final piece of the puzzle if you will :)

Reply:
we would run the tests against the remote schema, but it proved too flaky :/